test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -80,13 +80,16 @@ public void ValidateBasicException(InProcessServer server, string requestPath, E
             Assert.NotEmpty(actual.Context.Operation.Id);
         }
 
-#if NET451
         public void ValidateBasicDependency(string assemblyName, string requestPath, Func<IWebHostBuilder, IWebHostBuilder> configureHost = null)
         {
             DependencyTelemetry expected = new DependencyTelemetry();
             expected.ResultCode = "200";
             expected.Success = true;
+#if NET451
             expected.Name = requestPath;
+#else
+            expected.Name = "GET " + requestPath;
```

Comment:
if app is NET451 and uses AspNetCore, the HttpClient used is NET45 implementation, that could be instrumented with event source and profiler. Here we get only event source.